runtime/lua/vim/lsp.lua

```diff
@@ -1967,7 +1964,7 @@ end
 --  Unlike `buf_request`, this will collect all the responses from each server instead of handling them.
 --  A map of client_id:request_result will be provided to the callback
 --
----@returns (function) A function that will cancel all requests which is the same as the one returned from `buf_request`.
+---@return fun() # A function that will cancel all requests which is the same as the one returned from `buf_request`.
```

Comment:
```suggestion
---@return fun() # A function that will cancel all requests of the same type as the one returned from `buf_request`.
```

Reply:
Ended up removing the "of the same type as the one returned from `buf_request`." part altogether. I don't think it adds any clarity here but rather confuses.